web-app/src/common/prediction.ts

```diff
@@ -2,23 +2,46 @@ import { GameState, StoneColor } from './gameState';
 import { Coordinates, deepCloneObject } from './utils';
 import { getNbMovesNeededToWin } from './pathfinding';
 
+interface PlayPrediction {
+  coordinates: Coordinates;
+  playerRemainingMovesToWin: number;
+  opponentRemainingMovesToWin: number;
+  score: number;
+}
+
 export function getNextPlaySuggestion(gameState: GameState, stoneColor: StoneColor): Coordinates {

```

Comment:
Just a formating matter, but I find it a little too dense, if you add some blank line (in a logical way, like blank lines around the conditional throw) it can ease the reading:
```
  const playableCells = getPlayableCells(gameState);
  
  if (playableCells.length === 0) throw new Error("There is no playable cell in the given board.")

  const playPredictions = playableCells.map(coordinates =>
    getPlayPrediction(gameState, coordinates, stoneColor)
  );
  return getBestPossiblePlay(playPredictions).coordinates;
  
```